apps/communications/contacts/js/views/details.js

```diff
@@ -153,11 +154,25 @@ contacts.Details = (function() {
   };
 
   var showEditContact = function showEditContact() {
+    stopNFC();
     Contacts.showForm(true, contactData);
   };
 
   var setContact = function cd_setContact(currentContact) {
     contactData = currentContact;
+    startNFC(currentContact);
+  };
+
+  var startNFC = function(contact) {
+    if ('mozNfc' in navigator) {
```

Comment:
To avoid the race condition loading the nfc class, we could make startNFC and stopNFC promise based:
```
function startNFC = function(contact) {
   return new Promise((resolve)=>{
      if (!('mozNfc' in navigator)) { return resolve() } else {
        LazyLoader.load('/contacts/js/nfc.js', resolve);
      } 
   });
}
```

(didn't check the syntax) and same for stopnfc